docs/containers-authfile.5.md

```diff
@@ -0,0 +1,43 @@
+% containers-authfile(5)
+
+# NAME
+containers-authfile - syntax for the registry authentication file
+
+# DESCRIPTION
+
+A credentials file stored at `${XDG_RUNTIME_DIR}/containers/auth.json` in
+json format used to authenticate against container image registries.
+
+## FORMAT
+
+The auth.json file stores encrypted authentication information for the
+user to container image registries.  The file can have zero to many entries and
+is created by a `login` command from a container tool such as `podman login` or
+`buildah login`.  Each entry includes the name of the registry and then an auth
```

Comment:
Somehow related: I'd love there to be a `skopeo login` :)

Reply:
@QiWang19 Could you add this to your list of Things to Do.  Add skopoe login/logout.

Reply:
https://github.com/containers/skopeo/issues/839